packages/platforms/mellanox/x86-64/x86-64-mlnx-msn2700/onlp/builds/src/module/src/sysi.c

```diff
@@ -59,7 +59,21 @@ static char arr_cplddev_name[NUM_OF_CPLD][30] =
 const char*
 onlp_sysi_platform_get(void)
 {
-    return ONL_PLATFORM_NAME;
+    return "x86-64-mlnx-msn2700-all";
```

Comment:
The SYSI implementation allows for multiple platform support. If the platform identifier returned by platform_get() does not match the current platform name then an attempt to call sysi_platform_set() is called with the current platform name. If the driver supports the platform it should do whatever customization is necessary to change to the correct platform personality.